src/components/settings-menu/DevSection.js

```diff
@@ -69,7 +74,14 @@ const DevSection = () => {
         label="‍💻 Copy dev seeds"
         onPress={() => Clipboard.setString(DEV_SEEDS)}
       />
-      <ListItem label="‍👾 Connect to ganache" onPress={connectToGanache} />
+      <ListItem
+        label="‍👾 Connect to ganache"
+        onPress={async () => {
```

Comment:
Instead of declaring inline functions, use `useCallback` as it's done a few lines above for wipeKeychain